### PR TITLE
Add sentence embeddings in NLP section

### PR DESCRIPTION
**Pull request by revolunet**

### CODE REVIEW

**Comment by ildoc** [comment]
i think you should read [this](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) first...

File: readme.md [hunk 1]
```diff
@@ -308,6 +308,7 @@
 	- [Spanish](https://github.com/dav009/awesome-spanish-nlp#readme)
 	- [NLP with Ruby](https://github.com/arbox/nlp-with-ruby#readme)
 	- [Question Answering](https://github.com/seriousmac/awesome-qa#readme) - The science of asking and answering in natural language with a machine.
+	- [Sentence embeddings](https://github.com/Separius/awesome-sentence-embedding) - Pretrained sentence and word embedding models
```
Comment (ildoc) [inline readme.md]: should be

```suggestion
	- [Sentence embeddings](https://github.com/Separius/awesome-sentence-embedding#readme) - Pretrained sentence and word embedding models
```

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by sindresorhus** [comment]
// @Separius

**Pull request closed without merging**